src/components/podcast/Podcast.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import Episode from './episode/Episode';
+import PodcastHeader from './Header'
+import parsePodcastFeedFromXML from '../../util/podcast-util';
+
+/**
+ * Component for displaying the details about a podcast including the episodes
+ */
+const Podcast = (props) => {
+
+    // Track the podcast object
+    const [podcast, setPodcast] = useState({});
+    // Track the episodes separately
+    const [episodes, setEpisodes] = useState([])
+    // Track if the podcast has been loaded
+    const [loaded, setLoaded] = useState(false);
+
+    useEffect(() => { loadFeed(props.feed + '.xml') });
+
+    /**
+     * If the feed hasn't been loaded go out and load it.  Set the podcast
+     * state and loaded to true.
+     */
+    async function loadFeed(feedFile) {
```

Comment:
Generally speaking I don't see people loading data in components under the components/ directory of a project because usually that means those components are meant to be reusable which might mean rendering a bunch of them on a page... I realize that's not the case with this component, but I thought I'd call it out anyway. I think for the purposes of this app, this whole component should just be merged into screens/podcast/Detail.js